### ISSUE TITLE
eslint-config-airbnb-typescript-prettier@1.4.0 has incorrect peer dependency typescript@>=3.2.1 <3.7.0

### ISSUE THREAD

**Issue by binary64** [toshi-toma/eslint-config-airbnb-typescript-prettier#20]
during `yarn`:-
```
warning " > eslint-config-airbnb-typescript-prettier@1.4.0" has incorrect peer dependency "typescript@>=3.2.1 <3.7.0".
```

**Comment by toshi-toma** [comment]
Thank you for register the issue.

I updated some dependency and release [v1.4.1](https://github.com/toshi-toma/eslint-config-airbnb-typescript-prettier/blob/master/CHANGELOG.md#141-2019-11-21).
This warning showing?

**Comment by binary64** [comment]
Thank you
